core/trino-main/src/main/java/io/trino/connector/informationschema/InformationSchemaMetadata.java

```diff
@@ -217,18 +217,10 @@ private Set<QualifiedTablePrefix> getPrefixes(ConnectorSession session, Informat
         }
```

Comment:
I understand removing `tight coupling between the method implementation and its
caller`, however I do not follow the correctness issue. Are you saying it was also incorrect? If so in what cases?

Reply:
it looked incorrect. it wasn;'t actually incorrect (yet), jiust error-prine